.github/workflows/nightly-trivy-scan.yml

```diff
@@ -0,0 +1,25 @@
+name: Trivy Nightly Scan
+on:
+  schedule:
+    - cron: '0 2 * * *' # run at 2 AM UTC
+
+jobs:
+  nightly-scan:
+    name: Trivy Scan nightly
+    strategy:
+      fail-fast: false
+      matrix:
+        # maintain the versions of harbor that need to be actively
+        # security scanned
+        versions: [dev, v2.6.2, v1.10.14]
+        # list of images that need to be scanned
+        images: [harbor-core, harbor-db, harbor-exporter, harbor-jobservice, harbor-log, harbor-portal, harbor-registryctl, prepare]
```

Comment:
@Vad1mo I need help with the versions and images that we need to here. I updated them based on the latest updated images that I could find on docker hub

Reply:
Also, Do we need to have separate jobs for each docker image(the version will still be a matrix), or a single job that uses the above matrix.

The plus about having separate job for each image is that, it provides further option to configure the versions available. For example. I saw that harbor-exporter does not have a 1.10.14 while all other images have that version